steve/util.py

```diff
@@ -206,7 +371,17 @@ def get_project_config():
     :returns: config file
 
     """
-    return SteveConfig().load()
+    try:
+        return SteveConfig().load()
+    except ConfigNotFound as e:
+        try:
+            for cfg in [IniConfig, YamlConfig]:
+                if cfg == SteveConfig:
+                    continue  # already tried
+                return cfg().load()
+        except ConfigNotFound:
+            pass
+    raise e   # original raised error
```

Comment:
This tries to load the "default" config file format (set by assignment to SteveConfig) and if that fails tries all the other formats. In this case there is only one, but if you for some reason set SteveConfig = IniConfig, you don't have to change the code here.  The raise could indeed be indented with the same semantics.